draftivist_ui/src/components/campaign/CampaignIssue.tsx

```diff
@@ -90,7 +90,7 @@ export default function (): BaseComponent<Attrs> {
                         Next
                 </a>
                 <Dialog id="source">
-                    <span>These statements were provided by the <strong>{campaign.organizer}</strong>.</span>
+                    <span>These statements were written and verified by the {campaign.organizer}.</span>
```

Comment:
I wonder if "the" before `{campaign.organizer}` should be removed. Feels like that could cause issues if an organization's name already begins with "the"